src/main/java/gov/usgs/locator/LocInput.java

```diff
@@ -118,6 +119,12 @@ public LocInput(final Detection detection, JSONObject locationConfig) {
       UseSVD = true;
     }
 
+    if ((locationConfig != null) && (locationConfig.containsKey("ReassessInitialPhaseIDs"))) {
+      ReassessInitialPhaseIDs = (boolean) locationConfig.get("ReassessInitialPhaseIDs");
+    } else {
+      ReassessInitialPhaseIDs = true;

```

Comment:
same question just confirming if the default value should be true or false?

Reply:
See above